### PR TITLE
Clang compilation

### PR DESCRIPTION
**Pull request by clero**
Depends on log rework: https://github.com/01org/parameter-framework/pull/58

Add clang support to parameter-framework project.
Clang comes in Android with its standard library which supports c++11.

### CODE REVIEW

File: parameter/ParameterMgrPlatformConnector.cpp [hunk 1]
```diff
@@ -187,10 +186,18 @@ bool CParameterMgrPlatformConnector::isStarted() const
...
-        _pLogger->log(bIsWarning, strLog);
+        _pLogger->info(log);
+    }
+}
+
+void CParameterMgrPlatformConnector::warning(const string& log)
+{
+    if (_pLogger) {
+
+        _pLogger->info(log);
```
Comment (dawagner) [inline parameter/ParameterMgrPlatformConnector.cpp]: you meant `_pLogger->warning`, right ?

File: parameter/ParameterMgrLogger.h [hunk 2]
```diff
@@ -45,9 +45,14 @@ class CParameterMgrLogger : public CParameterMgr::ILogger
...
-    virtual void log(bool bIsWarning, const std::string& strLog)
+    virtual void info(const std::string& log)
     {
-        _parameterMgrConnector.doLog(bIsWarning, strLog);
+        _parameterMgrConnector.info(log);
+    }
+
+    virtual void warning(const std::string& log)
+    {
+        _parameterMgrConnector.info(log);
```
Comment (dawagner) [inline parameter/ParameterMgrLogger.h]: you meant `_parameterMgrConnector.warning`, right ?

File: parameter/ParameterMgrPlatformConnector.cpp (hunk 1 above)
Comment (clero) [inline parameter/ParameterMgrPlatformConnector.cpp]: done

File: parameter/ParameterMgrLogger.h (hunk 2 above)
Comment (clero) [inline parameter/ParameterMgrLogger.h]: done (copy paste are so bad ..)

**Comment by clero** [comment]
rebased on functional tests

**Comment by dawagner** [comment]
please resubmit against "next".

**Comment by clero** [comment]
replaced by #94

**Pull request closed without merging**